frontend/style/main.css

```
@@ -145,4 +145,115 @@ a:focus-visible {
+.message-list {
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-sm);
+}
+
+/* Base bubble style shared by all messages */
+.message-bubble {
+  max-width: 75%;
+  padding: var(--space-sm) var(--space-md);
+  border-radius: var(--radius-lg);
+  word-wrap: break-word;
```

Stylelint (17.9.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)